usr/share/rear/prep/RSYNC/GNU/Linux/200_selinux_in_use.sh

```diff
@@ -43,7 +43,7 @@ case $(basename $BACKUP_PROG) in
 	(tar)
 		if tar --usage | grep -q selinux  ; then
```

Comment:
I think it is worth to extract this into a function, it appears several times.

Reply:
@schlomo Indeed might be a good idea to do that